Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -716,19 +717,22 @@ extension OverviewViewController: DcrlibwalletSyncProgressListenerProtocol {
     func onSyncCanceled(_ willRestart: Bool) {
         DispatchQueue.main.async {
             self.updateUI(syncCompletedSuccessfully: false)
+            UIApplication.shared.isIdleTimerDisabled = false
```

Comment:
We have many different synchronous processes, so we have to check them all to complete to set UIApplication.shared.isIdleTimerDisabled = false